.gitignore

```diff
@@ -1,65 +1,9 @@
-# Built application files
```

Comment:
Try to make one thing first and then pull request instead of doing everything in one because first it's hard to read and then if i have some comments its hard to make changes.